src/openqaoa-azure/tests/test_devices_azure.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+import json
+import itertools
+import subprocess
+import pytest
+
+import openqaoa
```

Comment:
unused import

Reply:
This import is needed otherwise `openqaoa_azure.backends` results in an import error. (Circular import)